mantidproject/mantid
# Edit Release Notes 6.0 - Spec + LSS

**Description of work.**

This PR formats the release notes for Muons, LSS(Refl, SANS) and Spec(Direct/Indirect), for Release 6.0

**To test:**

build docs and check that the language in the changed files reads consistently and is clear.

Fixes #30373

*This does not require release notes* because **it is a change to release notes**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
